src/services/pronote/menu.ts

```diff
@@ -8,12 +8,20 @@ export const getMenu = async (account: PronoteAccount, date: Date): Promise<Menu
   }
 
   const menu = await pronote.menus(account.instance, date);
-  if (!menu.days || menu.days.length === 0) {
+  if (!menu.days?.length) {
     return { date };
   }
+
   const day = menu.days[0];
 
-  if (day.date?.getTime() !== date.getTime()) {
+  const updatedDate = new Date(day.date);
+  updatedDate.setDate(updatedDate.getDate() + 1);
```

Comment:
Est-ce qu'en faisant ceci, ça a une influence ?

```suggestion
  updatedDate.setUTCDate(updatedDate.getUTCDate() + 1);
```